dns/zone-configs/k8s.io.yaml

```diff
@@ -0,0 +1,216 @@
+---
+# The top-level k8s.io zone
+'':
+  - type: A
+    value: 23.236.58.218 # Our vanity redirector, hosted in GKE
+  - type: MX
+    values:
+    - exchange: aspmx.l.google.com.
+      preference: 1
+    - exchange: alt1.aspmx.l.google.com.
+      preference: 5
+    - exchange: alt2.aspmx.l.google.com.
+      preference: 5
+    - exchange: alt3.aspmx.l.google.com.
+      preference: 10
+    - exchange: alt4.aspmx.l.google.com.
+      preference: 10
+  - type: TXT
+    values:
+    - google-site-verification=RJbZ_ganmSWvslSKOBG-QHv62XTjJZcigpWIFttStFs
+    - v=spf1 include:_spf.google.com ~all
+www:
+  type: CNAME
+  value: k8s.io.
+_acme-challenge:
```

Comment:
this was probably from a LetsEncrypt run that wasn't properly deleted; I think we can remove it from this file.